docs/conf.py

```diff
@@ -44,7 +44,13 @@ def setup(app) -> None:
         indextemplate="pair: %s; configuration value",
     )
 
-    font_dirs: list[str] = []
+    font_dirs: list[str] = [
+        str(weblate_dir / font_dir)
+        for font_dir in (
+            "weblate/static/js/vendor/fonts/font-source/",
+            "weblate/static/vendor/font-kurinto/",
+        )
+    ]
```

Comment:
Seems like a wrong merge, you should adjust `font_locations` above.

Reply:
Uh true, it should be an empty list. Thanks!